packages/edit-post/src/components/layout/index.js

```diff
@@ -99,7 +100,13 @@ function Layout( {
 				<div className="edit-post-layout__metaboxes">
 					<MetaBoxes location="advanced" />
 				</div>
+				{ isMobileViewport && sidebarIsOpened && <ScrollLock /> }
 			</div>
+			{ isRichEditingEnabled && mode === 'visual' && (
+				<div className="edit-post-layout__footer">
```

Comment:
It needs to have a role defined, the same way as the container to make the navigation possible for people who use keyboard only. In this case, `role="region"` might be too general. `contentinfo` might be a better fit: 

> The `contentinfo` landmark role is used to identify information repeated at the end of every page of a website, including copyright information, navigation links, and privacy statements. This section is commonly called a footer.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role

If that would be a good fit, `footer` can be used instead of the `div`. In that case, the role is going to be satisfied by the HTML tag.

Reply:
There is also this question of how such block breadcrumb would be announced when using VoiceOver.

Reply:
I think it needs to be a region like header/sidebar and other regions we have. I did that in #18043 I can add that quickly here as well.